src/Lexer.js

```diff
@@ -75,14 +75,14 @@ const tokenRegex = new RegExp(
     "([ \r\n\t]+)|" +                                 // whitespace
     "([!-\\[\\]-\u2027\u202A-\uD7FF\uF900-\uFFFF]" +  // single codepoint
     "|[\uD800-\uDBFF][\uDC00-\uDFFF]" +               // surrogate pair
-    "|\\\\(?:[a-zA-Z]+|[^\uD800-\uDFFF])" +           // function name
+    "|\\\\(?:[a-zA-Z@]+|[^\uD800-\uDFFF])" +          // function name
```

Comment:
Incidentally, the regex change won't affect `@{...}` support in `tabular` (the only use I know of `@` as an active char), nor will it affect the most typical usage of `\@`, namely, `\@.`.  What it would break is if you used `\@` immediately followed by a letter, which ... I can't imagine doing.  So the other option is to leave this change as is for now.